src/table/table.stories.ts

```diff
@@ -156,27 +156,27 @@ storiesOf("Components|Table", module).addDecorator(
 				<h4 ibmTableHeaderTitle>{{title}}</h4>
 				<p ibmTableHeaderDescription>{{description}}</p>
 			</ibm-table-header>
-			<ibm-table-toolbar [model]="model" [batchText]="batchText">
+			<ibm-table-toolbar [model]="model" [batchText]="batchText" #toolbar>
```

Comment:
It would be ideal if we could handle this switching (semi) automatically ... This is still a viable pattern for other components that might be passed in, but for `ibmButton`s we could just set the `tabindex`s automatically

Reply:
I like the idea, but how/where do we set the `tabindex`s automatically for `ibmButton`s? Inside the `ibmButton` directive? Hopefully the consumers don't want to control `tabindex` themselves, otherwise we'll have conflict.

On a side note, I just checked the Carbon React stories, and it seems they did the same thing in story code and not in the data table code:

https://github.com/carbon-design-system/carbon/blob/1a4cdb1051fb587f406aa082c99717c2ba6f3c75/packages/react/src/components/DataTable/stories/with-batch-actions.js#L63

Reply:
@cal-smith any thoughts on this?

Reply:
Sorry! This works for now 👍 we can investigate if there's an easier solution later!